frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx

```diff
@@ -235,7 +240,16 @@ export const OidcAuth = () => {
                         />
                     </Grid>
                 </Grid>
-
+                <ConditionallyRender
+                    condition={ssoSyncShown}
+                    show={
+                        <SsoGroupSettings
+                            ssoType={'OIDC'}
+                            data={data}
+                            setValue={setValue}
+                        ></SsoGroupSettings>
```

Comment:
```suggestion
                        />
```

Reply:
I suspect these kind of things can be done in autoformatter/linter config, we should make a plan to include these